modules/registry-operator-config-resources-secret-openstack.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * registry/configuring-registry-operator.adoc
+
+
+[id="registry-operator-config-resources-secret-openstack_{context}"]
+= Configuring a Secret for the Image Registry Operator
```

Comment:
Secret -> secret
As per guidelines [1]. Throughout PR

[1] https://github.com/openshift/openshift-docs/blob/enterprise-4.1/contributing_to_docs/term_glossary.adoc#secret

Reply:
Thanks for the catch. I went ahead and made the change for the all of the registry storage content for consistency. 

There are other instances of "Secret" that we'll need to scrub. It might also be a discussion point of whether or not we're capitalizing Kubernetes objects consistently.